docs/tasks/access-kubernetes-api/setup-api-aggregator.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Setup the API Aggregator
+assignees:
+- chenopis
+---
+
+{% capture overview %}
+
+Setting up the API Aggregator will allow the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
+
+{% endcapture %}
+
+{% capture prerequisites %}
```

Comment:
Ok. Are there any other prereqs then that they'll need before they can get started?